src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/PowCoinviewRule.cs

```diff
@@ -155,13 +128,15 @@ public virtual void ExecuteBlock(RuleContext context, TaskScheduler taskSchedule
                 bool passed = checkInputs.All(c => c.GetAwaiter().GetResult());
```

Comment:
Didn't we want to convert this to async?

Reply:
I didn't know we wanted to do this, but I will.

Reply:
Well we can yes, we will be changing behaviour from how it was before.
In fact I think the script verification should go in its own rule but I did not want too many changes in behaviour.